src/core/ext/transport/chttp2/transport/internal.h

```diff
@@ -38,6 +38,7 @@
 #include "src/core/lib/iomgr/combiner.h"
 #include "src/core/lib/iomgr/endpoint.h"
 #include "src/core/lib/iomgr/timer.h"
+#include "src/core/lib/support/abstract.h"
```

Comment:
can we move this code into a separate polymorphic_manual_constructor_test program?

Reply:
Done